rules/falco_rules.yaml

```diff
@@ -2876,7 +2876,10 @@
 # - k8s.gcr.io/fluentd-gcp-scaler in GCP/GKE
 - macro: user_known_k8s_client_container
   condition: >
-    (k8s.ns.name="kube-system" and container.image.repository=k8s.gcr.io/fluentd-gcp-scaler) or
+    (k8s.ns.name="kube-system" and (
+      container.image.repository=k8s.gcr.io/fluentd-gcp-scaler or
+      container.image.repository=k8s.gcr.io/node-problem-detector/node-problem-detector
```

Comment:
can we make a list called `user_known_k8s_client_images` within which are:
`[k8s.gcr.io/fluentd-gcp-scaler, k8s.gcr.io/node-problem-detector/node-problem-detector]`
So that in the macro, we can update with `(k8s.ns.name="kube-system" and container.image.repository in (user_known_k8s_client_images))`